mlos_bench/mlos_bench/launcher.py

```diff
@@ -50,7 +51,15 @@ class Launcher:
 
     def __init__(self, description: str, long_text: str = "", argv: Optional[List[str]] = None):
         _LOG.info("Launch: %s", description)
-        parser = argparse.ArgumentParser(description=f"{description} : {long_text}")
+        epilog = textwrap.dedent("""
```

Comment:
```suggestion
        epilog = """
```
I don't think we ever need to call `textwrap.dedent()` for argparser help strings. it should automatically strip and dedent the text

Reply:
Ahh, I didn't realize that `argparser` did that too.  That might be why the newlines aren't preserved here.

Anyways, removed for now.  Can try and reprettify that text output some other time.